File: 2853/2853.c
int highestSalariesDifference(struct TreeNode* root) {
    if (!root) return 0;
    int l = highestSalariesDifference(root->left);
    int r = highestSalariesDifference(root->right);
    return 1 + (l > r ? l : r);
}

File: 2853/2853.cs
public class Solution {
    public int HighestSalariesDifference(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(HighestSalariesDifference(root.left), HighestSalariesDifference(root.right));
    }
}